examples/requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy
 tyro
-Pillow
\ No newline at end of file
+Pillow
+pytorch3d @ git+https://github.com/facebookresearch/pytorch3d@v0.7.5
```

Comment:
It seems like only the tests rely on p3d, it would be nice to remove as a dependency since it's a pretty big one and if we only need it for matrix transforms there's more common lightweight options like scipy.

Reply:
Unfortunately I needed the matrix transforms to be differentiable so that the gradients can flow into the Euler angle and position parameters. I tried implementing my own transformations, but ran into some instability in training. I have been using `se3_exp_map` and `se3_log_map`, but if you have any replacements I am open to try them.